modules/adding-node-iso-flags.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// *nodes/nodes/nodes-nodes-adding-node-iso.adoc
+
+[id="adding-node-iso-flags{context}"]
+= Adding a node with command flags
+
+You can add a single node to your cluster by using command flags to specify configurations for the new node.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`)
+* You have an active connection to your target cluster
+* You have a kubeconfig file available
+
+.Procedure
+
+. Generate the ISO image by running the following command. The MAC address must be specified using a command flag.
++
+[source,terminal]
+----
+$ oc adm node-image create --kubeconfig='<path_to_kube_config>' --mac-address=<mac_address>
+----
++
+--
+where:
+
+`<path_to_kube_config>`:: Specifies the path to the kubeconfig file.
```

Comment:
Is this the case with running just about any `oc` command? If so, I don't see this flag included really anywhere else in the docs, and maybe it's an implicit expectation that users know to either define it elsewhere, or because they'll immediately see an error message if they don't have one.

In either case, if it's something that will be obvious to users if they don't include this flag (or have it defined elsewhere), I can drop this from the command and let it be an implicit requirement